elastio-stream-kafka/elastio_stream_kafka.py

```diff
@@ -0,0 +1,76 @@
+import argparse
+import logging
+import os
+import subprocess
+import ast
+import json
+import time
+
+from kafka import KafkaProducer
+
+
+logging.basicConfig(level=logging.CRITICAL)
+
+parser = argparse.ArgumentParser(
+    prog="Elastio stream kafka",
+)
+subparser = parser.add_subparsers(dest="mod")
+# subparser accept two posible modes of work this script backup and restore
+
+backup_parser = subparser.add_parser("backup")
+# backup mode arguments
+backup_parser.add_argument("--topic_name", required=True, type=str, nargs="?", help="Enter Kafka topic name to backup.")
+backup_parser.add_argument("--brokers", required=True, type=str, nargs="+", help="Enter one or more Kafka brokers separated by spaces.")
+backup_parser.add_argument("--vault", required=True, type=str, nargs="?", help="Enter vault name.")
+
+restore_parser = subparser.add_parser("restore")
+# restore mode arguments
+restore_parser.add_argument("--rp_id", required=True, type=str, nargs="?", help="Enter recovery point ID.")
+restore_parser.add_argument("--topic_name", required=True, type=str, nargs="?", help="Enter Kafka topic name to restore data for this topic.")
+restore_parser.add_argument("--brokers", required=True, type=str, nargs="+", help="Enter one or more Kafka brokers separated by spaces.")
+
+args = parser.parse_args()
+
+if args.mod == "backup":
+    print("Backup started.")
+    res = os.popen(f"python consumer.py --topic_name {args.topic_name} --brokers {' '.join(args.brokers)} | elastio stream backup --stream-name {args.topic_name} --vault {args.vault} --output-format json").read()
```

Comment:
The stream is named after a topic, but what if someone has two Kafka clusters with the same topic and wants to back up them both? It's certainly the situation in Blue stack, where we have similar clusters in several environments.

Reply:
The stream name should probably just be a parameter.